src/src/secret.py

```diff
@@ -1,32 +1,50 @@
+from dotenv import load_dotenv
+import os
+
+# Load environment variables from a .env file
+load_dotenv()
+
+
 def getGithubKey():
-    return 'GHKEYXXX'
+    return os.environ.get(key="ONLINE_TOOL_GITHUB_KEY")
+
 
 def getGithubSecret():
-    return 'GHSECRETXXX'
+    return os.environ.get(key="ONLINE_TOOL_GITHUB_SECRET")
+
 
 def getSecretKey():
-    return 'DJANGOSECRETXXX'
+    return "NON_EMPTY_DJANGO_SECRET" if os.environ.get(key="DJANGO_SECRET_KEY") == "" \
+        else os.environ.get(key="DJANGO_SECRET_KEY", failobj="NON_EMPTY_DJANGO_SECRET")
```

Comment:
I found `os.getenv(key,default)` to do the same thing this big if-else is doing. Should we change it to that?

Reply:
@rtgdk actually the environment variable is holding an empty string `""` which is provided by GitHub to outside PRs, on the other hand, I observed os module utilizes default value only when the environment variable is `None` and not for `""` (empty string). 
I feel this is the only fix, thoughts?

Reply:
@kmehant Makes sense. Merging.